<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 240, 225);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>941</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 110, 64);
border-radius: 10px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DATA KENDARAAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CUCI MOTOR</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 61, 89);
border-radius: 10px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="kodeKendaraan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 193, 59);
border-radius: 5px</string>
    </property>
    <property name="placeholderText">
     <string>  Masukkan ID Kendaraan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ID Kendaraan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Jenis Kendaraan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 0);
border-radius: 10px;
font: 5 8pt &quot;Arial Black&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Cari</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../praktikum/python code/Perpustakaan/Asset/cari.png</normaloff>../../../../praktikum/python code/Perpustakaan/Asset/cari.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(0, 159, 60);
font: 5 8pt &quot;Arial Black&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Reset inputan</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../praktikum/python code/Perpustakaan/Asset/reset.png</normaloff>../../../../praktikum/python code/Perpustakaan/Asset/reset.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="jenisKendaraan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 193, 59);
border-radius: 5px</string>
    </property>
    <property name="placeholderText">
     <string>  Masukkan Jenis Kendaraan</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 193, 59);
border-radius:10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(68, 196, 255);
font: 5 8pt &quot;Arial Black&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../praktikum/python code/Perpustakaan/Asset/save.png</normaloff>../../../../praktikum/python code/Perpustakaan/Asset/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 5 8pt &quot;Arial Black&quot;;
color:white;
background-color: rgb(95, 238, 114);</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../praktikum/python code/Perpustakaan/Asset/update.png</normaloff>../../../../praktikum/python code/Perpustakaan/Asset/update.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: 5 8pt &quot;Arial Black&quot;;
background-color: rgb(255, 86, 74);
color:white;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../praktikum/python code/Perpustakaan/Asset/delete.png</normaloff>../../../../praktikum/python code/Perpustakaan/Asset/delete.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="gridKendaraan">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
